### DISCUSSION TITLE
Is it possible to configure svelte-kit package to compile to vanilla JS?

### DISCUSSION THREAD

**Discussion by NazimHAli** [sveltejs/kit#3011] [Q&A] [answered]
Hello,

I've been testing out `sveltekit package` following the docs https://kit.svelte.dev/docs#packaging.  Using the example of the component `src/lib/Foo.svelte` exported by `src/lib/index.js`, running `svelte-kit package` copies the component to `packages/Foo.svelte`. I was expecting it to be compiled JS so that it could be packaged and consumed in any framework/project.

Is this the wrong expectation for packaging? Or can we configure it to package compiled JS?

**Comment by ignatiusmb** [comment, marked as answer]
No, it's not the wrong expectation, but it's mentioned in the docs that it doesn't do any bundling, yet. There is, however, an issue for this on #2572 that you can follow

**Reply by NazimHAli** [reply]
Awesome, thank you for the confirmation! I missed that part while reading the docs.
